libvast/src/format/csv.cpp

```diff
@@ -13,12 +13,6 @@
 
 #include "vast/format/csv.hpp"
```

Comment:
Are you going to squash these commits? If not I'd suggest editing the title of this one to prominently mention that it actually fixes the CSV parser, not only the tests.

Reply:
Reworded it, thanks.